src/geode/geosciences/representation/io/cross_section_input.cpp

```diff
@@ -25,6 +25,19 @@
 
 #include <geode/geosciences/representation/core/cross_section.h>
 
+namespace
+{
+    void add_to_message( std::string& message,
```

Comment:
If you take message by reference, use absl::StrAppend() to avoid creating a new string